com.usebottles.bottles.yml

```diff
@@ -0,0 +1,69 @@
+---
+app-id: com.usebottles.bottles
+runtime: org.gnome.Sdk
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+add-extensions:
+  org.gnome.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '3.38'
+command: bottles
+finish-args:
+- --allow=multiarch
+- --share=network
+- --device=dri
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --socket=pulseaudio
+- --socket=session-bus
+- --filesystem=~/Games
+- --filesystem=~/wineprefixes
+- --filesystem=~/.Bottles
```

Comment:
It should use `--persist`.

Reply:
This one is used to obtain wineprefixes from old Bottles version. We used a different path (.Bottles) in v1, now we use `.local/share/bottles` we don't really need to map it.